code/campaign/src/main/java/com/adobe/marketing/mobile/campaign/AlertMessage.java

```diff
@@ -92,17 +99,20 @@ public void onShow() {
          * Calls the {@link #viewed()} method implemented in the parent class {@link CampaignMessage}.
          */
         @Override
-        public void onDismiss() {
+        public void onDismiss(final @NonNull Presentable<Alert> presentable) {
             viewed();
         }
 
+        @Override
+        public void onHide(final @NonNull Presentable<Alert> presentable) {}
+
         /**
          * Invoked when an error occurs when showing the alert.
          * <p>
          */
         @Override
-        public void onError(final UIError uiError) {
-            Log.debug(CampaignConstants.LOG_TAG, SELF_TAG, "Error occurred when attempting to display the alert message: %s.", uiError.toString());
+        public void onError(final @NonNull Presentable<Alert> presentable, final @NonNull PresentationError presentationError) {
+            Log.debug(CampaignConstants.LOG_TAG, SELF_TAG, "Error occurred when attempting to display the alert message");
```

Comment:
we should print the presentationError details if possible

Reply:
I did ask @prudrabhat about this but core does not expose the details of the presentationError right now. And none of our extensions take any action based on the error, so guessing it's fine to just print there was some error for now?